### ISSUE TITLE
Raven Ruby not sending errors.

### ISSUE THREAD

**Issue by vinay-nadig** [getsentry/sentry-ruby#202]
I have the following configuration:

``` ruby
# config/initializers/raven.rb
require 'raven'

logger = ::Logger.new("log/raven.log")
logger.level = ::Logger::DEBUG

Raven.configure do |config|
  config.logger = logger
  config.excluded_exceptions = []
  config.current_environment = Rails.env
  config.environments = %w[ development ]
  config.ssl_verification = false
  config.dsn = 'http://#:#@fqdn/#'
end
```

``` ruby
# Gemfile
gem "sentry-raven", :git => "https://github.com/getsentry/raven-ruby.git"
```

Output of `rake raven:test`

``` bash
rake raven:test
Client configuration:
-> server: <server_fqdn>
-> project_id: 2
-> public_key: #
-> secret_key: #

Sending a test event:
-> event ID: 9cd2f904a16647b5be435841e6b436c1

Done!
```

When I run the rake task, the exception is being logged in the application.

However, no other errors are being logged.

I have a dummy route and action:

``` ruby
# routes.rb
get   '/error'     => 'payslip#error',    as: :payslip_error
```

``` ruby
# payslip_controller.rb
def error
  logger.info Raven.client.configuration.send_in_current_environment?
  1/0
end
```

The logger logs a `true` in the log file but the exception is never sent to the server.
If I replace the `1/0` with `Raven.capture { 1/0 }`, the exception is sent to the server.

I have also tried setting the `$SENTRY_DSN` value manually. But, no luck their either.

``` bash
export SENTRY_DSN=http://#:#@fqdn/2
```

Command used to run the rails server: `rails s` (Running in the the development environment locally)

Am I missing something obvious?

Rails Version: 4.1.4

Let me know if you need any other information.

Thanks.

**Comment by vladCovaliov** [comment]
I had a similar problem because raven.rb wasn't added in config/initializers. :dragon_face:

**Comment by shrirambalakrishnan** [comment]
This issue exists even now. As mentioned here, setting the `config.consider_all_requests_local = false` worked.

**Comment by nateberkopec** [comment]
@shrirambalakrishnan please open a new issue.

**Comment by avdept** [comment]
I had an issue when I set development as env that sends error, but spring has cached initial initializer code, hence it worked only after `spring stop`
